Scripts/flow.php

```diff
@@ -20,6 +20,10 @@
 }
 
 if (isset($argv[1]) && ($argv[1] === 'typo3.flow:core:setfilepermissions' || $argv[1] === 'flow:core:setfilepermissions' || $argv[1] === 'core:setfilepermissions')) {
+    $perms = decoct(fileperms(__DIR__. '/setfilepermissions.sh') & 0777);
+        if ($perms != '700'){
```

Comment:
Sorry about the indentation. As for the permissions, setfilepermissions.sh sets it's own mode to 700: "sudo chmod 700 $0" on line 76 so why would we allow it to be anything else?